src/tipsy.py

```diff
@@ -0,0 +1,13 @@
+from board import Board
+
+board = Board()
+
+while True:
+    board.display()
+    input_command = ''
+
+    while (input_command.lower() not in [Board.EAST, Board.WEST, Board.NORTH, Board.SOUTH]):
```

Comment:
We'd prefer to have an "nerly empty" main file, call features inside dedicated feature files.
I would be nice to have the user commands inside their own file

Reply:
Also not fan of the n,s,e,w commmands. For later, maybe think about something more fluent to have a better user experience (like arrows for exemple)

Reply:
We opted for the n,s,e,w commands to avoid dealing with arrows, which represent a special difficulty in the terminal.

Reply:
I'm adding instruction as explain in the trello. It will be clearer.

Reply:
done